packages/engine/src/worker/runner/python/runner.py

```diff
@@ -149,7 +149,7 @@ def handle_runner_error(self, exc, tb):
         # Rust process that a runner error occurred so it immediately knows
         # that the runner exited.
 
-        error = "Runner error: " + str(traceback.format_exception(type(exc), exc, tb))
+        error = "Runner error: " + str(traceback.format_exception(*exc_info))
```

Comment:
Can we either pass the destructured args, or can we just type hint?

Or at least a comment saying it comes from the function so you can destructure